### PR TITLE
Recaptcha Changes

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Committing the changes for Recaptcha Story for fixing QA issues

### CODE REVIEW

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
     String searchFieldDefaultValue = "store_id";
     String searchFieldValueDefaultValue = "2";
     String servicePathCartFetchItems = "/us/V1/guest-carts/";
+    String recaptchaSiteKey = "6LfrFKQUAAAAAMzFobDZ7ZWy982lDxeps8cd1I2i";
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java]: @nitheshkumar-hcl  This is magento service config, Create separate common configuration

File: core/src/main/java/com/hcl/ecomm/core/models/RecaptchaSiteKey.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+@Model(adaptables = Resource.class)
+public class RecaptchaSiteKey {
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/models/RecaptchaSiteKey.java]: @nitheshkumar-hcl  rename this and append Model as this sling model class.

File: core/src/main/java/com/hcl/ecomm/core/services/CustomerService.java [hunk 3]
```diff
@@ -12,6 +12,8 @@
 
 	public String customerProfileServicePath();
 
+	public String getRecaptchaSiteKey();
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/CustomerService.java]: @nitheshkumar-hcl  put this method in common service

File: core/src/main/java/com/hcl/ecomm/core/services/impl/CustomerServiceImpl.java [hunk 4]
```diff
@@ -71,6 +71,9 @@ public String customerProfileServicePath() {
         return config.customerService_profilePath();
     }
 
+    @Override
+    public String getRecaptchaSiteKey() { return  config.getRecaptchaSiteKey_string();}
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/CustomerServiceImpl.java]: @nitheshkumar-hcl  Move this in common service impl

File: core/src/main/java/com/hcl/ecomm/core/models/RecaptchaSiteKey.java (hunk 2 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/models/RecaptchaSiteKey.java]: Committed the changes as per the review comments

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java (hunk 1 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java]: Committed the changes as per the review comments

File: core/src/main/java/com/hcl/ecomm/core/services/CustomerService.java (hunk 3 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/CustomerService.java]: Committed the changes as per the review comments

File: core/src/main/java/com/hcl/ecomm/core/services/impl/CustomerServiceImpl.java (hunk 4 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/CustomerServiceImpl.java]: Committed the changes as per the review comments

**Pull request merged**